libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/MapboxRouteLineUtilsTest.kt

```diff
@@ -306,7 +259,7 @@ class MapboxRouteLineUtilsTest {
                 styleLayerExists(RouteConstants.ALTERNATIVE_ROUTE2_TRAFFIC_LAYER_ID)
             } returns false
             every { styleLayerExists(RouteConstants.WAYPOINT_LAYER_ID) } returns false
-            every { styleLayerExists("mapbox-location-foreground-layer") } returns true
+            // every { styleLayerExists("mapbox-location-foreground-layer") } returns true
```

Comment:
can we remove?

Reply:
I made adjustments such that the layer initialization will first check for the existence of the specified belowLayerId. If the layer doesn't exist a check will be made for the existence of LocationComponentConstants.MODEL_LAYER. If that layer doesn't exist null will be used and the route line will be placed at the top of the layer stack.  

What do you think?